packages/core/test/menu/menuTests.tsx

```diff
@@ -96,6 +96,26 @@ describe("MenuItem", () => {
         assert.strictEqual(wrapper.text(), "Object");
     });
 
+    it("popover can be controlled with popoverProps", () => {
+        // Ensures that popover props are passed to Popover component, except content property
+        const popoverProps = {
+            content: "CUSTOM_CONTENT",
+            inline: false,
+            interactionKind: PopoverInteractionKind.CLICK,
+            popoverClassName: "CUSTOM_POPOVER_CLASS_NAME",
+        };
+        const wrapper = shallow(
+            <MenuItem iconName="style" text="Style" popoverProps={popoverProps}>
+                <MenuItem text="one" />
+                <MenuItem text="two" />
+            </MenuItem>,
+        );
+        assert.strictEqual(wrapper.find(Popover).prop("inline"), popoverProps.inline);
+        assert.strictEqual(wrapper.find(Popover).prop("interactionKind"), popoverProps.interactionKind);
+        assert.notStrictEqual(wrapper.find(Popover).prop("popoverClassName").indexOf(popoverProps.popoverClassName), 0);
+        assert.notStrictEqual(wrapper.find(Popover).prop("content"), popoverProps.content);
```

Comment:
Check `inline`, `interactionKind` (some of overwriteable props), `content` (shouldn't be passed) and `popoverClassName` (custom class should me included)